rosdep/python.yaml

```
@@ -8580,6 +8580,17 @@ python3-sqlalchemy-utils:
   nixos: [python3Packages.sqlalchemy-utils]
   opensuse: [python3-SQLAlchemy-Utils]
   ubuntu: [python3-sqlalchemy-utils]
+python3-sqlmodel:
+  debian:
+    '*':
+      packages: [python3-sqlmodel]
+    bullseye:
+      pip:
+        packages: [sqlmodel]
+    buster: null
+  ubuntu:
+    pip:
+      packages: [sqlmodel]
```

It also looks like this is available in the latest Ubuntu (https://packages.ubuntu.com/search?keywords=python3-sqlmodel&searchon=names&suite=all&section=all), so this can be:

```suggestion
  ubuntu:
    '*': [python3-sqlmodel]
    focal: null
    jammy: null
```